release-notes/unreleased/main.yml

```diff
@@ -0,0 +1,6 @@
+issue_key:
```

Comment:
Missing issue key. The name of the release note is also wrong (`main.yml`)

Reply:
done.